lib/WebService/Async/SmartyStreets.pm

```diff
@@ -0,0 +1,183 @@
+package WebService::Async::SmartyStreets;
+# ABSTRACT: Access SmartyStreet API
+
+use strict;
+use warnings;
+
+our $VERSION = '0.001';
+
+=HEAD
+
+WebService::Async::SmartyStreets - calls the SmartyStreets API and checks for the validity of the address
+
+=head1 VERSION
+
+version 0.001
+
+=head1 SYNOPSIS
+    
+    {
```

Comment:
Simpler just to move this inline, the `get_smartystreets` extra sub overcomplicates this.